apps/dotcom/client/index.html

```diff
@@ -1,34 +1,77 @@
+		/>
+		<script type="application/ld+json">
+			{
+				"@context": "https://schema.org",
+				"@type": "SoftwareApplication",
+				"name": "tldraw",
+				"url": "https://www.tldraw.com/",
+				"applicationCategory": "BusinessApplication",
+				"operatingSystem": "WEB",
+				"description": "A free and instant collaborative whiteboarding tool.",
+				"creator": {
+					"@type": "Organization",
+					"name": "tldraw"
+				}
+			}
+		</script>
+		<style>
+			#root {
+				position: relative;
+				inset: 0;
+				width: 100%;
+				height: 100%;
+				overflow: hidden;
+			}
 
+			.visually-hidden {
+				position: absolute !important;
+				clip: rect(1px, 1px, 1px, 1px);
+				height: 1px;
+				width: 1px;
+				overflow: hidden;
+				white-space: nowrap;
+				pointer-events: none;
+				user-select: none;
+				-webkit-user-select: none;
+			}
+		</style>
 		<meta name="referrer" content="strict-origin-when-cross-origin" />
 		<meta name="theme-color" content="#FFFFFF" data-rh="true" />
 		<meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover" />
 		<link rel="icon" type="image/svg+xml" href="/favicon.svg" />
-		<meta name="application-name" content="tldraw" />
+		<link rel="apple-touch-icon" href="/apple-touch-icon.png" />
+		<link rel="apple-touch-icon" sizes="152x152" href="/apple-touch-icon-152x152.svg" />
+		<link rel="apple-touch-icon" sizes="167x167" href="/apple-touch-icon-167x167.svg" />
+		<link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon-180x180.svg" />
+
 		<meta name="apple-mobile-web-app-capable" content="yes" />
 		<meta name="apple-mobile-web-app-status-bar-style" content="default" />
 		<meta name="apple-mobile-web-app-title" content="tldraw" />
 
+		<meta name="application-name" content="tldraw" />
+		<meta name="title" content="tldraw" />
 		<meta name="description" content="A free and instant collaborative whiteboarding tool." />
+		<meta name="image" content="https://www.tldraw.com/social-twitter.png" />
+
 		<meta name="format-detection" content="telephone=no" />
 		<meta name="mobile-web-app-capable" content="yes" />
 		<meta name="msapplication-config" content="browserconfig.xml" />
 		<meta name="msapplication-TileColor" content="#FFFFFF" />
 		<meta name="msapplication-tap-highlight" content="no" />
 
-		<link rel="apple-touch-icon" href="/apple-touch-icon.png" />
-		<link rel="apple-touch-icon" sizes="152x152" href="/apple-touch-icon-152x152.svg" />
-		<link rel="apple-touch-icon" sizes="167x167" href="/apple-touch-icon-167x167.svg" />
-		<link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon-180x180.svg" />
-
 		<!-- $PRELOADED_FONTS -->
-		<link rel="preconnect" href="https://fonts.googleapis.com" />
-		<link rel="preconnect" href="https://fonts.gstatic.com" />
+		<link rel="preconnect" href="https://fonts.googleapis.com" crossorigin="anonymous" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonymous" />
```

Comment:
looks like, judging from their docs it should be like this:
https://fonts.google.com/knowledge/using_type/using_web_fonts_from_a_font_delivery_service
```
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```

Reply:
I read some suggestions that anonymous improves things. Possible cargo culting